src/layouts/shared/ErrorBoundary/ErrorBoundary.test.tsx

```diff
@@ -1,20 +1,20 @@
 import * as Sentry from '@sentry/browser'
 import { render, screen } from '@testing-library/react'
 import { MemoryRouter, Route } from 'react-router-dom'
-import { vi } from 'vitest'
```

Comment:
the changes in this file are to convert to TS

Reply:
Nice!